airflow/contrib/example_dags/example_gcp_bigtable_operators.py

```diff
@@ -0,0 +1,147 @@
+CLUSTER_ZONE = getenv('CBT_CLUSTER_ZONE', 'europe-west1-b')
+CLUSTER_NODES = getenv('CBT_CLUSTER_NODES', '3')
+CLUSTER_NODES_UPDATED = getenv('CBT_CLUSTER_NODES_UPDATED', '5')
+CLUSTER_STORAGE_TYPE = getenv('CBT_CLUSTER_STORAGE_TYPE', '2')
+TABLE_ID = getenv('CBT_TABLE_ID', 'some-table-id')
+POKE_INTERVAL = getenv('CBT_POKE_INTERVAL', '60')
+# [END howto_operator_gcp_bigtable_args]
+
+default_args = {
+    'start_date': airflow.utils.dates.days_ago(1)
+}
+
+with models.DAG(
+    'example_gcp_bigtable_operators',
+    default_args=default_args,
+    schedule_interval=datetime.timedelta(days=1)
+) as dag:
+    # [START howto_operator_gcp_bigtable_instance_create]
+    create_instance_task = BigTableInstanceCreateOperator(
+        project_id=PROJECT_ID,
+        instance_id=INSTANCE_ID,
+        main_cluster_id=CLUSTER_ID,
+        main_cluster_zone=CLUSTER_ZONE,
+        instance_display_name=INSTANCE_DISPLAY_NAME,
+        instance_type=int(INSTANCE_TYPE),
+        instance_labels=json.loads(INSTANCE_LABELS),
+        cluster_nodes=int(CLUSTER_NODES),
+        cluster_storage_type=int(CLUSTER_STORAGE_TYPE),
+        task_id='create_instance',
+    )
+    # [END howto_operator_gcp_bigtable_instance_create]
+
+    # [START howto_operator_gcp_bigtable_cluster_update]
+    cluster_update_task = BigTableClusterUpdateOperator(
+        project_id=PROJECT_ID,
+        instance_id=INSTANCE_ID,
+        cluster_id=CLUSTER_ID,
+        nodes=int(CLUSTER_NODES_UPDATED),
+        task_id='update_cluster',
+    )
+    # [END howto_operator_gcp_bigtable_cluster_update]
+
+    # [START howto_operator_gcp_bigtable_instance_delete]
+    delete_instance_task = BigTableInstanceDeleteOperator(
+        project_id=PROJECT_ID,
+        instance_id=INSTANCE_ID,
+        task_id='delete_instance',
+    )
+    # [END howto_operator_gcp_bigtable_instance_delete]
+
+    # [START howto_operator_gcp_bigtable_table_create]
+    create_table_task = BigTableTableCreateOperator(
+        project_id=PROJECT_ID,
+        instance_id=INSTANCE_ID,
+        table_id=TABLE_ID,
+        task_id='create_table',
+    )
+    # [END howto_operator_gcp_bigtable_table_create]
+
+    # [START howto_operator_gcp_bigtable_table_wait_for_replication]
+    wait_for_table_replication_task = BigTableTableWaitForReplicationSensor(
+        project_id=PROJECT_ID,
+        instance_id=INSTANCE_ID,
+        table_id=TABLE_ID,
+        poke_interval=int(POKE_INTERVAL),
+        task_id='wait_for_table_replication',
+    )
+    # [END howto_operator_gcp_bigtable_table_wait_for_replication]
+
+    # [START howto_operator_gcp_bigtable_table_delete]
+    delete_table_task = BigTableTableDeleteOperator(
+        project_id=PROJECT_ID,
+        instance_id=INSTANCE_ID,
+        table_id=TABLE_ID,
+        task_id='delete_table',
+    )
+    # [END howto_operator_gcp_bigtable_table_delete]
+
+    dag >> wait_for_table_replication_task >> delete_table_task
```

Comment:
Are you sure we want dag >> in here? I thought we should have just task dependencies?